lock-impl/src/main/java/com/palantir/lock/impl/LockServiceImpl.java

```diff
@@ -467,23 +468,25 @@ private void logNullResponse(LockClient client, LockRequest request, @Nullable H
                 token == null ? UnsafeArg.of("lockToken", "null") : UnsafeArg.of("lockToken", token));
     }
 
-    private void logLockAcquisitionFailure(Map<LockDescriptor, LockClient> failedLocks) {
+    @VisibleForTesting
```

Comment:
We don't really test log lines specifically, and I don't think we need to here either.

Reply:
Is there a reason why we do it for `logSlowLockAcquisition` ? (see `LockServiceImplTest`)